include/zephyr/drivers/emul.h

```diff
@@ -8,6 +8,10 @@
 #ifndef ZEPHYR_INCLUDE_DRIVERS_EMUL_H_
 #define ZEPHYR_INCLUDE_DRIVERS_EMUL_H_
 
+#include <zephyr/drivers/i2c_emul.h>
```

Comment:
Instead of including:

Could do forward-declaration of:
		struct i2c_emul;
		struct espi_emul;
		struct spi_emul;